java/vector/src/main/java/org/apache/arrow/vector/types/pojo/Schema.java

```diff
@@ -99,21 +101,38 @@ public static Schema convertSchema(org.apache.arrow.flatbuf.Schema schema) {
   private final Map<String, String> metadata;
 
   public Schema(Iterable<Field> fields) {
-    this(fields, null);
+    this(fields, (Map<String, String>) null);
   }
 
   /**
```

Comment:
nit: seems like this doc comment got moved away from the constructor it's supposed to target.

Reply:
thanks @lidavidm just fixed.

Reply:
Ah, that set off checkstyle, sorry :( (looks like it still wants a docstring for the public constructor)

Reply:
thats what I get for commiting w/o building ;-) 

fixed again.

Reply:
Thanks! I'll merge once CI is green.